s/features/xio-components/id/xio-id.css.ts

```diff
@@ -11,15 +11,19 @@ button {
 .container {
 	position: relative;
 	display: flex;
-	flex-direction: row;
+	flex-direction: var(--_flex-direction);
```

Comment:
- let's leave the styling of xiome-id untouched, i think we want to keep the copy button on the right side